tests/test_seasonal_water_yield_regression.py

```diff
@@ -738,7 +739,7 @@ def test_climate_zones_regression(self):
         make_agg_results_csv(agg_results_csv_path, climate_zones=True)
 
         SeasonalWaterYieldRegressionTests._assert_regression_results_equal(
-            os.path.join(args['workspace_dir'], 'aggregated_results_cz.shp'),
+            os.path.join(args['workspace_dir'], 'aggregated_results_swy_cz.shp'),

```

Comment:
Ah, this is a really interesting example that might help us decide on an approach for the filename!

If we wanted to go with the DRY approach, we would _also_ want this string here to be updated as well ... but doing so would also hide the meaning a bit behind some string manipulations:

```python
'_cz'.join(os.path.splitext(AGGREGATED_RESULTS_FILENAME))
```
Is a lot harder to read (as a human, anyways :) ) than `aggregated_results_swy_cz.shp`

So for the sake of explicitness, I might suggest we use the static strings rather than a module-level variable.  But again, what do you think about this reasoning?

Reply:
Sounds good, I'll revert it to static strings. Agree that ideally all usages of the string would be replaced with the variable, which seems impractical here.